plugins/database/src/mid_level_routines.cpp

```diff
@@ -1381,9 +1381,14 @@ int checkObjIdByTicket( const char *dataId, const char *accessLevel,
     char original_collection_name[MAX_NAME_LEN];
     std::vector<std::string> bindVars;
     bindVars.push_back( dataId );
+    bindVars.push_back(dataId);
```

Comment:
spacing is different.    and we want to push `dataId` twice?   I guess once as `coll_id`...

Reply:
> spacing is different.

Yeah, I kept changing my mind on that. I figured it was better to show the actual fix rather than make everything line up, but the change is tiny so I think cleaning up both lines is fine too.

Will update.

> and we want to push `dataId` twice? I guess once as `coll_id`...

Correct. The query is attempting to find a collection name. The problem was that it was only considering collections that contained data objects. The fix was to also consider empty collections.

Reply:
Done